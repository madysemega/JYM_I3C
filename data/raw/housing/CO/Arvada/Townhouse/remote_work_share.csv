predictions_remote_work_share
4.836653625965118
3.444323759078979
4.288666162490844
3.2782828843593594
2.971265046596527
2.803390848636627
3.254975650310516
2.83328821182251
2.8492695188522337
2.7500646138191223
2.2942949616909027
2.3175554120540616
3.0993619632720946
2.029556038379669
3.7184944725036617
2.300461847782135
2.4067014265060425
2.7931716990470883
2.469351749420166
1.6959304273128508
1.8977371311187743
1.863673630952835
2.0409085476398467
1.7447879242897033
2.5578358244895933
1.8303495121002196
2.9819918489456176
