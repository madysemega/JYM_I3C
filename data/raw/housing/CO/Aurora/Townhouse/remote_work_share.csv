predictions_remote_work_share
7.416603181362152
7.102643320560455
7.212981972694397
8.518449465036392
6.775884560346603
7.628938434123993
6.60110454916954
7.0607484793663025
8.53050532102585
7.873895082473755
8.995835087299348
8.402289835214615
7.1689987933635715
7.572453949451447
6.926073036193848
7.613143817186356
6.841987172365188
9.426994273662567
7.158593360185623
7.219398138523102
9.121775767803193
6.853915891647339
8.752339569330216
6.70466964006424
7.2579318618774415
7.474189517498017
7.512446365356445
