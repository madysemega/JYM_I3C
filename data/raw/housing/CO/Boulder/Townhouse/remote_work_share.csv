predictions_remote_work_share
15.757180324196817
16.343408550024034
18.58747675776482
16.68214541554451
16.707673704624177
16.053315294981005
14.575482082366944
16.40397445201874
15.796721592545511
14.803237928748132
13.32896942138672
15.385054431557657
15.191904425621034
15.999808562994005
15.691581897139551
16.93947147846222
17.2126667290926
16.700807695984842
15.509505099058153
15.125072658061983
16.390883895754815
14.246020061373713
14.080239058732989
14.517513012886049
15.798880454897882
16.264605942368508
16.080802054405215
