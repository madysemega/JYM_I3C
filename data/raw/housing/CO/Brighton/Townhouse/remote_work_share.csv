predictions_remote_work_share
9.781594358682632
4.539379937052726
9.107690883278845
9.61441874206066
5.296836855411529
8.673220372796058
9.991239202022552
8.2083241045475
7.99165544450283
10.519927389621733
5.81307207763195
8.268323705792426
11.521518684625624
7.17599169909954
10.339780150651931
9.465537623763083
9.139553055167198
8.875238994956016
8.319132451415062
11.162067457437514
8.391596569418907
6.963208345770836
13.246317633390426
12.217432737350464
8.913249388933181
12.592977347373962
8.643472103476524
