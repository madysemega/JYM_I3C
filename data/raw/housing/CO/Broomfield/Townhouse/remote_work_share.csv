predictions_remote_work_share
12.289421024322511
10.81631356418133
18.217072465419776
17.864186906218535
12.662942131757738
15.025513582825663
12.366055809855462
15.284237378239634
19.460592457056052
15.446632304191592
12.023982267975809
12.428899829983713
11.699884458780291
10.780217965841295
12.295841585993768
13.026857441067698
11.869558610916139
12.789257814884188
16.412895658016208
13.49881625711918
13.481157412528994
11.501710822582247
13.261725696325303
12.401656653881075
12.488221281170846
12.257528793215753
15.825935086011889
