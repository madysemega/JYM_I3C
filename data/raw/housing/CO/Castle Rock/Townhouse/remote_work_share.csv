predictions_remote_work_share
4.371129460334777
4.847618346214294
5.263330626487732
3.987335410118103
4.2980802583694455
3.126714596748352
4.3597540330886835
3.6267885351181026
4.298343653678893
3.6665700292587275
3.4286306238174435
4.395643506050109
4.642165265083313
4.171481232643127
5.102080559730529
4.439010968208312
4.156484966278076
3.7125484228134153
3.8695056200027462
3.261419630050659
4.216546883583068
4.628602437973022
4.237784738540649
4.65544641494751
4.422319221496582
4.817592973709106
4.499802131652832
