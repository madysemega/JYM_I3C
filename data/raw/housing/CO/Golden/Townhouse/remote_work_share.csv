predictions_remote_work_share
9.7973062479496
9.685604240894317
10.818911652565001
10.36208196878433
9.481909236907958
11.262879351377485
9.936231663227082
9.703960069417953
10.557082688808439
10.06564793586731
8.76891527056694
10.91147246956825
11.030612673759459
10.661233426332473
10.926809756755826
10.229986371994018
10.971658939123152
11.292978731393813
10.498003591299055
9.105944751501083
10.462455842494963
10.98036428451538
10.226321351528167
10.632095161676405
10.283238554000855
11.73942910671234
11.252663506269453
